### PR TITLE
Remove resources requirement in PipelineInterface

### PR DESCRIPTION
**Pull request by vreuter**
This should solve https://github.com/epigen/looper/issues/28

### CODE REVIEW

File: looper/models.py [hunk 1]
```diff
@@ -1126,24 +1127,40 @@ def as_data_frame(self):
         return _pd.DataFrame([s.as_series() for s in self.samples])
 
 
-    def to_csv(self, path):
+    def write(self, path, sep=None):
```
Comment (nsheff) [inline looper/models.py]: should we keep a `to_csv` as a wrapper of `write` for backwards compatibility?

Comment (vreuter) [inline looper/models.py]: I'd prefer to have just `write`, but only slightly. It's totally up to you; I'll add it back if you'd like.

Comment (vreuter) [inline looper/models.py]: Agree, good call

Comment (nsheff) [inline looper/models.py]: I will go ahead and merge this, we can add back in if need be.

Comment (vreuter) [inline looper/models.py]: Sounds good.

**Pull request merged**